cmdstanpy/stanfit/mcmc.py

```diff
@@ -129,6 +129,10 @@ def __getattr__(self, attr: str) -> np.ndarray:
             # pylint: disable=raise-missing-from
             raise AttributeError(*e.args)
 
+    def __getstate__(self) -> dict:
```

Comment:
Would you mind adding a docstring or comment just explaining what this is used for/why it is here?